src/common/trackerConfig.js

```diff
@@ -156,9 +156,9 @@ export const getTextSize = (ctx, markerSize, text, fontSize) => {
 /**
  * @ignore
  */
-export const getDelayColor = (delayInMs, cancelled) => {
+export const getDelayColor = (delayInMs, cancelled, isText) => {
```

Comment:
What does `isText` mean?

Reply:
It is when we want to retrieve the color for the delay text, not the background.

Reply:
Describe the parameters of the functions.